### ISSUE TITLE
When passing option multiple: true to .click() and not all el's clickable, 'undefined' err thrown.

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#90]
When using the command `.click()` and passing the option `multiple: true`, I can see that at least 1 of the elements I am attempting to click on is not visible. The click command eventually times out and in the command log displays error:

```
 CypressError: Timed out retrying: undefined
```

**Code**
```
cy.get(".close").click(multiple: true)
```

It would be nice to instead get an error about "one or more of the elements not being visible, so not being clickable"

**Additional Info**
<img width="584" alt="screen shot 2015-12-08 at 10 55 16 am" src="https://cloud.githubusercontent.com/assets/1271364/11660233/376d97ae-9d9a-11e5-9df0-4bd9aad15eac.png">

**Comment by brian-mann** [comment]
`{multiple: true}` doesn't have anything to do with this. `undefined` is actually the bug. Normally Cypress tells you exactly why it can't click the element: ie what is covering it up. The problem here is that for some reason it could not figure that out. That is the bug.

**Comment by brian-mann** [comment]
It is likely that [`0.13.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0133-12252015) fixed this. Can you try your scenario again and see if you can still consistently get this message to appear?

**Comment by jennifer-mann** [comment]
This is still occurring in [`0.13.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0133-12252015).

**Comment by brian-mann** [comment]
Fixed in [`0.13.41](https://github.com/cypress-io/cypress/wiki/Changelog#0134-12312015)
